scripting/get5_apistats.sp

```diff
@@ -123,10 +123,13 @@ static Handle CreateRequest(EHTTPMethod httpMethod, const char[] apiMethod, any:
   Handle req = SteamWorks_CreateHTTPRequest(httpMethod, formattedUrl);
   if (StrEqual(g_APIKey, "")) {
     // Not using a web interface.
+    LogDebug("Failed to create request because no ApiKey was provided");
```

Comment:
Make this a `LogError` instead and make it more descriptive of what is actually missing. Put in the name of the Cvar so people know what to adjust.